jenkins_pipelines/environments/manager-4.0-dev-acceptance-tests-NUE

```diff
@@ -20,7 +20,7 @@ node('sumaform-cucumber') {
             string(name: 'terracumber_ref', defaultValue: 'master', description: 'Terracumber Git ref (branch, tag...)'),
             booleanParam(name: 'terraform_init', defaultValue: false, description: 'Call terraform init (needed if modules are added or changes)'),
             choice(name: 'rake_namespace', choices: ['cucumber', 'parallel'], description: 'Choose [parallel] (Clients and some features will run in parallel) or [cucumber] (all sequential)'),
-            choice(name: 'functional_scope', choices: ['default', 'smdba', 'spacecmd', 'spacewalk_utils', 'visualization', 'notification_message', 'virtual_host_manager',  'subscription_matching', 'formulas', 'sp_migration', 'cve_audit', 'onboarding', 'content_lifecycle_management', 'res', 'recurring_actions', 'maintenance_windows', 'cluster_management', 'building_container_images', 'kubernetes_integration', 'openscap', 'ubuntu', 'action_chains', 'salt_ssh', 'tomcat', 'changing_software_channels', 'monitoring', 'salt', 'cobbler', 'sumatoolbox', 'virtualization', 'hub', 'retail', 'configuration_channels', 'content_staging', 'proxy', 'traditional_client', 'xmlrpc', 'power_management'], description: 'Choose a functional scope and your job will only run tests of that scope')
+            extendedChoice(name: 'functional_scopes', defaultValue: 'default', multiSelectDelimiter: ',', quoteValue: false, saveJSONParameterToFile: false, type: 'PT_CHECKBOX', visibleItemCount: 30, value: 'default,scope_smdba,scope_spacecmd,scope_spacewalk_utils,scope_visualization,scope_notification_message,scope_virtual_host_manager,scope_subscription_matching,scope_formulas,scope_sp_migration,scope_cve_audit,scope_onboarding,scope_content_lifecycle_management,scope_res,scope_recurring_actions,scope_maintenance_windows,scope_cluster_management,scope_building_container_images,scope_kubernetes_integration,scope_openscap,scope_ubuntu,scope_action_chains,scope_salt_ssh,scope_tomcat,scope_changing_software_channels,scope_monitoring,scope_salt,scope_cobbler,scope_sumatoolbox,scope_virtualization,scope_hub,scope_retail,scope_configuration_channels,scope_content_staging,scope_proxy,scope_traditional_client,scope_xmlrpc,scope_power_management,scope_retracted_patches', description: 'Choose the functional scopes that you want to test')
```

Comment:
nitpick, spaces after the commas please?

Reply:
No, we can't this is implementation design of this plugin.